qemu/tests/cfg/multi_host.cfg

```diff
@@ -33,27 +37,6 @@
                 - fd:
                 - exec:
 
-            variants booting_phase:
-                #Time when start migration
-                - after_login_vm:
-                    paused_after_start_vm = no
-                    need_to_login = yes
-                    preprocess_env = no
-                - early_boot_vm:
-                    no measure_migration_speed
```

Comment:
Could you help explain why remove this line and not add it in the following code?